welcome/index.adoc

```diff
@@ -4,4 +4,39 @@
+
+To navigate the {product-title} documentation, you can use one of the following methods:
+
+* Use the left navigation bar to browse the documentation.
+* Select the task that interests you from the contents of this Welcome page.
+
+== Installation activities
+Explore these {product-title} installation tasks.
+
+- *xref:../installing/install-ocp-operator.adoc#installing-using-an-operator[Installing using an Operator]*: Learn how to install {product-title} on {ocp} by using the {product-title} Operator.
+- *xref:../installing/installing_helm/install-helm-quick.adoc#install-helm-quick[Installing quickly using Helm charts]*: Learn how to install {product-title} by using Helm charts.
+- *xref:../installing/install-quick-roxctl.adoc#install-quick-roxctl[Installing quickly by using the `roxctl` CLI]*: Learn how to install {product-title} by using the `roxctl` command-line interface (CLI). `roxctl` is a CLI for running commands on {product-title}.
+
+== Operating {product-title}
+Explore various activities you can perform by using {product-title}.
+
+- *xref:../operating/view-dashboard.adoc#view-dashboard[Viewing the dashboard]*: Find information about {product-title} real-time interactive dashboard. Learn how to use it to view key metrics from all your hosts, containers, and services.
+- *xref:../operating/manage-compliance.adoc#manage-compliance[Managing compliance]*: Understand how to run automated checks and validate compliance based on industry standards, including CIS, NIST, PCI, and HIPAA.
+- *xref:../operating/manage-vulnerabilities.adoc#manage-vulnerabilities[Managing vulnerabilities]*: Learn how to identify and prioritize vulnerabilities for remediation.
+- *xref:../operating/respond-to-violations.adoc#respond-to-violations[Responding to violations]*: Learn how to view policy violations, drill down to the actual cause of the violation, and take corrective actions.
+
+== Configuring {product-title}
+Explore these {product-title} typical configuration tasks.
+
+- *xref:../configuration/add-custom-certificates.adoc#add-custom-certificates[Adding custom certificates]*: Learn how to use a custom TLS certificate with {product-title}. After you set up a certificate, users and API clients do not have to bypass the certificate security warnings.
+- *xref:../backup_and_restore/backing-up-acs.adoc#configure-authentication[Backing up {product-title}]*: Learn how to perform manual and automated data backups for {product-title} and use these backups for data restoration in the case of an infrastructure disaster or corrupt data.
+- *xref:../configuration/configure-automatic-upgrades.adoc#configure-automatic-upgrades[Configuring automatic upgrades for secured clusters]*: Stay up-to-date by automating the upgrade process for each secured cluster.
+
+== Integrating with other products
+Learn how to integrate {product-title} with other products.
```

Comment:
```suggestion
Learn how to integrate {product-title} with the following products:
```

Reply:
Do you want to leave these xrefs in or comment them out until the integration files are published?